README.md

```diff
@@ -20,10 +20,20 @@ Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C
   cd ..
   catkin_make
   ```
+  
+### Application can work at two modes:
+-  Local mode
+  -  All the ROS nodes run at the same machine.
+-  Network mode
+  -  ROS nodes run at two different machine: Server and Client.
+  -  Client runs ROS nodes "RVIZ" and "point_cloud_generator_node".
+  -  Server runs all other ROS nodes except two nodes running on client.
```

Comment:
What is the robot node(simulated of course),is something like RobotStudio to simulate real robot ?
What we changed so far is to implement network function on "full simulation mode", I thought the RobotStudio will act like real robot for us? ( didn't try RobotStudio yet, not sure my understanding is right or not).
After we do some study and testing to how to deal with real robot (or RobotStudio) and real senor Kinect2, we can implement network function to support other work mode below.

> Run blending demo in robot simulation mode (simulated robot and real sensor data)
> roslaunch godel_irb2400_support irb2400_blending.launch sim_sensor:=false
> Run blending demo in sensor simulation mode (real robot and simulated sensor)
> roslaunch godel_irb2400_support irb2400_blending.launch sim_robot:=false robot_ip:=[robot ip]
> Run blending demo in full real mode
> roslaunch godel_irb2400_support irb2400_blending.launch sim_sensor:=false 
> sim_robot:=false robot_ip:=[robot ip]